definitions/consented/json/CaseEvent/CaseEvent.json

```diff
@@ -821,7 +821,7 @@
     "Name": "Solicitor Upload document",
     "Description": "Solicitor Upload document",
     "DisplayOrder": 48,
-    "PreConditionState(s)": "awaitingInfo",
+    "PreConditionState(s)": "awaitingInfo;readyForHearing",
```

Comment:
This seems wrong - as per requirement - this change need on Upload document event